operate/schema/src/main/java/io/camunda/operate/schema/opensearch/OpensearchSchemaManager.java

```diff
@@ -104,7 +104,37 @@ public void createSchema() {
   }
 
   @Override
-  public boolean setIndexSettingsFor(Map<String, ?> settings, String indexPattern) {
+  public void checkAndUpdateIndices() {
+    LOGGER.info("Updating Indices with currently-configured number of replicas...");
+    final String currentConfigNumberOfReplicas =
+        String.valueOf(operateProperties.getOpensearch().getNumberOfReplicas());
+    getIndexNames("*")
+        .forEach(
+            index -> {
+              final Map<String, String> indexSettings =
+                  getIndexSettingsFor(index, NUMBERS_OF_REPLICA, REFRESH_INTERVAL);
+              final String currentIndexNumberOfReplicas = indexSettings.get(NUMBERS_OF_REPLICA);
+              final String refreshInterval = indexSettings.get(REFRESH_INTERVAL);
```

Comment:
I don't see these lines related to fetching and setting the refresh interval in the elasticsearch schema manager, is there a reason to not do the same there?

Reply:
`setIndexSettings()` in opensearch requires the `REFRESH_INTERVAL` setting to be included and non-null or it throws an error, while elasticsearch does not require the setting. I can include it in elasticsearch if you feel it's best to keep them similar. Ref: [elasticsearch](https://github.com/camunda/zeebe/blob/4909920630988d6b24f2326d305f945e4aa0d9fc/operate/schema/src/main/java/io/camunda/operate/schema/elasticsearch/ElasticsearchSchemaManager.java#L86), [opensearch](https://github.com/camunda/zeebe/blob/4909920630988d6b24f2326d305f945e4aa0d9fc/operate/schema/src/main/java/io/camunda/operate/schema/opensearch/OpensearchSchemaManager.java#L110)